### PR TITLE
Deeptools pkg

### PR DESCRIPTION
**Pull request by bgruening**

### CODE REVIEW

File: packages/package_python_2_7_10_pybigwig_0_2_3/tool_dependencies.xml [hunk 1]
```diff
@@ -6,7 +6,7 @@
     <package name="python" version="2.7.10">
         <repository name="package_python_2_7_10" owner="iuc" prior_installation_required="True" />
     </package>
-    <package name="pybigwig" version="0.1.9">
+    <package name="pybigwig" version="0.2.3">
```
Comment (martenson) [inline packages/package_python_2_7_10_pybigwig_0_2_3/tool_dependencies.xml]: Do I understand it correctly that this is fixing an erorr?

Comment (bgruening) [inline packages/package_python_2_7_10_pybigwig_0_2_3/tool_dependencies.xml]: Yes, the version does not fit the package version, this was probably an oversight and this package is only in TTS so far.

**Comment by martenson** [comment]
looks great, looking forward to it!

**Comment by bgruening** [comment]
Thanks for merging @martenson!

**Pull request merged**